app/helpers/filters_helper.rb

```diff
@@ -14,6 +14,12 @@ def filter_selected?(filter, value, filter_type)
     return true if (selected_filters["owning_organisation"].present? || selected_filters["owning_organisation_text_search"].present?) && filter == "owning_organisation_select" && value == :specific_org
     return true if (selected_filters["managing_organisation"].present? || selected_filters["managing_organisation_text_search"].present?) && filter == "managing_organisation_select" && value == :specific_org
 
+    return true if !selected_filters.key?("user") && filter == "uploaded_by" && value == :all
+    return true if selected_filters["uploaded_by"] == "specific_user" && filter == "uploaded_by" && value == :specific_user
```

Comment:
Would it be at all nicer to refactor this method a bit?
How would it read if we were to extract `filter == "uploaded_by"? Should this entire method be a case statement based on filter?